client/state/analytics/actions/record.js

```diff
@@ -24,7 +26,7 @@ export const recordCustomFacebookConversionEvent = ( name, properties ) =>
 export const recordCustomAdWordsRemarketingEvent = ( properties ) =>
 	recordEvent( 'adwords', { properties } );
 
-export const recordPageView = ( url, title, service, properties = {}, options = {} ) => ( {
+export const recordPageViewBasic = ( url, title, service, properties = {}, options = {} ) => ( {
```

Comment:
I'd suggest not exporting this one, otherwise we risk someone to track page views without our new property we're introducing here. We can export it at some point if we need to, although I doubt we will.

Also, naming nit: I'd personally name it `recordPageViewBase` and not `recordPageViewBasic`, because `base` implies that it will be composed, which we're essentially doing a few lines below.